renderdoc/driver/ihv/intel/intel_gl_counters.cpp

```diff
@@ -104,6 +104,25 @@ void IntelGlCounters::addCounter(const IntelGlQuery &query, GLuint counterId)
   counter.desc.resultType = glToRdcCounterType(counter.dataType);
   counter.desc.unit = CounterUnit::Absolute;
 
+  if(counter.desc.description.contains("Unit: cycles."))
```

Comment:
I'm not a big fan of this kind of string compare especially when it's not spec'd and could change. At least this falls back to unitless, but is there any alternative to actually query the type as an enum from the driver?

Reply:
Yeah, I'm not happy about this either, but doing it properly would require adding a new extension with a new API and I don't think it's worth it for such a minor change. I think we'll need more things to fix/add for a new extension.

Reply:
Fair enough yeh, I thought that might be the case.